addons/web/static/src/js/view_form.js

```diff
@@ -6338,6 +6376,7 @@ instance.web.form.widgets = new instance.web.Registry({
     'one2many_list' : 'instance.web.form.FieldOne2Many',
     'reference' : 'instance.web.form.FieldReference',
     'boolean' : 'instance.web.form.FieldBoolean',
+    'toggle_button' : 'instance.web.form.FieldToggleBoolean',
```

Comment:
Mmmh ... As this is a widget for the boolean field, this place makes sense to me.